shims/spark310/src/main/scala/com/nvidia/spark/rapids/shims/spark310/Spark310Shims.scala

```diff
@@ -111,17 +111,17 @@ class Spark310Shims extends Spark301Shims {
         }),
     GpuOverrides.expr[AnsiCast](
       "Convert a column of one type of data into another type",
-      new CastChecks() {
+      new CastChecks {
         import TypeSig._
         // nullChecks are the same
 
         override val booleanChecks: TypeSig = integral + fp + BOOLEAN + STRING
         override val sparkBooleanSig: TypeSig = numeric + BOOLEAN + STRING
 
-        override val integralChecks: TypeSig = integral + fp + BOOLEAN + STRING
+        override val integralChecks: TypeSig = numeric + STRING + STRING
```

Comment:
`BOOLEAN + STRING` not `STRING + STRING`

Reply:
Fixed.